extensions/amp-list/0.1/amp-list.js

```diff
@@ -758,45 +758,43 @@ export class AmpList extends AMP.BaseElement {
 
   /**
    * Undoes previous size-defined layout, must be called in mutation context.
-   * @param {string} previousLayout
+   * @param {!Layout} previousLayout
    */
   undoPreviousLayout_(previousLayout) {
-    switch (previousLayout) {
-      case Layout.RESPONSIVE:
-        this.element.classList.remove('i-amphtml-layout-responsive');
-        setStyles(this.element, {
-          height: '',
-          width: '',
-        });
-        break;
-      case Layout.FLEX_ITEM:
-        setStyles(this.element, {
-          height: '',
-          width: '',
-        });
-        break;
-      case Layout.FIXED:
-        this.element.classList.remove('i-amphtml-layout-fixed');
-        setStyles(this.element, {
-          height: '',
-        });
-        break;
-      case Layout.FIXED_HEIGHT:
-        this.element.classList.remove('i-amphtml-layout-fixed-height');
-        setStyles(this.element, {
-          height: '',
-          width: '',
-        });
-        break;
-      case Layout.INTRINSIC:
-        this.element.classList.remove('i-amphtml-layout-intrinsic');
-        break;
-      default:
-        // fall through, always remove sizer and size-defined class
+    const {element} = this;
+    if (previousLayout == Layout.RESPONSIVE) {
```

Comment:
It seems like there were fallthrough wins we could do in the switch statement.

Reply:
Will revert to switch.